### ISSUE TITLE
sendText should return an object with message ID

### ISSUE THREAD

**Issue by mrbarnk** [open-wa/wa-automate-nodejs#490]
hi @smashah , can sendText return an object with the message ID. So we can be able to check the status of the message.

**Comment by smashah** [comment]
@mrbarnk what status do you want to check of the message? sendText already returns the message id string

**Comment by mrbarnk** [comment]
I want to check if the message has been read or delivered.

**Comment by smashah** [comment]
if it returns the whole message object to you then it will only be a snapshot of that message immediately after it has been sent which would be pretty useless no?

**Comment by mrbarnk** [comment]
yes,useless.

**Comment by mrbarnk** [comment]
If we can be able to getstatus of a message at anytime, would be better.

**Comment by smashah** [comment]
@github-actions run

<details>
<summary>⚡ Release!  ⚡</summary>

```js
(async () => {
function exec(cmd) {
  console.log(execSync(cmd).toString());
}

// Config
const gitUserEmail = "github-actions[bot]@users.noreply.github.com";
const gitUserName = "github-actions[bot]";

exec(`echo "//registry.npmjs.org/:_authToken=$NPM_TOKEN" > .npmrc`);
exec(`git config --global user.email "${gitUserEmail}"`);
exec(`git config --global user.name "${gitUserName}"`);
exec(`npm i`);
exec(`npm run release-ci`);

//comment on the issue
var result = execSync(`npx auto-changelog -o ./tempchangelog.txt --commit-limit false --template ./compact-keepachangelog.hbs --stdout`).toString();

    await postComment(result);

//create changelog image
exec(`npm run release-image`);
exec(`git commit -a -m 'updated release-image'`);
exec(`git push --force`);
  })();
```
</details>

**Comment by smashah** [comment]
# Changelog

## 🚀 Release [1.9.52](https://github.com/open-wa/wa-automate-nodejs/compare/1.9.51...1.9.52) (2020-05-27)

- Release 1.9.52 [`8b8ef68`](https://github.com/open-wa/wa-automate-nodejs/commit/8b8ef683e728c92a270d4ff1fb20c1ecfba76307)
- feature: implemented `getSingleProperty` #490 [`9554ceb`](https://github.com/open-wa/wa-automate-nodejs/commit/9554ceba68a6731b023f81b0677cfebf8814c320)

**Comment by smashah** [comment]
@mrbarnk use this.

https://open-wa.github.io/wa-automate-nodejs/classes/client.html#getsingleproperty

```javascript
const ack  = await client.getSingleProperty('Msg',"true_12345678912@c.us_9C4D0965EA5C09D591334AB6BDB07FEB",'ack')
```
